charts/fleet/templates/service.yaml

```diff
@@ -1,13 +1,14 @@
-{{- if .Values.metrics.enabled }}
```

Comment:
I think we still want this condition here. Why was it removed?

Reply:
An error TY